client/js/vector-drag.js

```diff
@@ -63,6 +62,13 @@ class VectorDrag extends Pointer {
           return feature
         })
       const element = evt.map.getTargetElement()
+      if (evt.dragging) {
+        this.cursor = 'grabbing'
+        element.style.cursor = 'grabbing'
+      } else {
+        this.cursor = 'crosshair'
+        element.style.cursor = 'pointer'
```

Comment:
i think some of the flickering is caused  by this
changing from 
element.style.cursor = 'pointer'
to
element.style.cursor = 'crosshair'
improves this.